nbs/99_manuscript/clustering/15-analyze_lvs_assoc.ipynb

```diff
@@ -0,0 +1,3850 @@
+{
```

Comment:
<p>where did you come up with number?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Ah good question. Fifteen final partitions (from k=5 to k=29) were selected using a method that measures how much information is retained from the ensemble. Here we focused on the partition with the maximum number of clusters (k=29).</p>

Reply:
<p>I will add a comment about this in the notebook.</p>